presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -58,6 +60,251 @@ General Properties
+    ``query.max-cpu-time`` will be cancelled.
+
+
+``query.max-execution-time``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``duration``
+    * **Default value:** ``100d``
+
+    Used as default for session property ``query_max_execution_time``. Unlike ``query.max-run-time`` which enforces
+    starting from the query creation time, this property does not consider the queueing time.
+
+
+``query.queue-config-file``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Default value:**
+
+    **Deprecated** The path to the queue config file. Queues are used to manage the number of concurrent queries across
+    the system. More information on queues and how to configure them can be found in :doc:/admin/queue.
+
+
+``query.remote-task.max-callback-threads``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Minimum Value:** ``1``
+    * **Default value:** ``1000``
+
+    This value describes the maximum size of the thread pool used to handle responses to HTTP requests for
+    each task. Increasing this value will cause more resources to be used for handling HTTP communication
+    itself, but may also improve response time when Presto is distributed across many hosts or there are
+    a lot of small queries being run.
+
+
+``query.remote-task.min-error-duration``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``duration``
+    * **Minimum Value:** ``1s``
+    * **Default value:** ``2m``
+
+    The minimal time that HTTP worker must be unavailable before the coordinator assumes the worker crashed.
+    A higher value may be recommended in unstable connection conditions. In order to consider a node dead,
+    the defined time must pass between two failed attempts of HTTP communication, with no successful communication
+    in between.
+
+
+``query.remote-task.max-error-duration``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``duration``
+    * **Minimum Value:** ``1s``
+    * **Default value:** ``5m``
+
+    The maximum time that HTTP worker can be unavailable before the coordinator assumes the worker crashed.
+    Use this property to increase the error tolerance for communication between coordinator and worker based on the
+    length of time that the query has run without errors. The node will be considered alive up to the defined time
+    before it hit failures.
+
+
+``query.schedule-split-batch-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Minimum Value:** ``1``
+    * **Default value:** ``1000``
+
+    The number of splits that will be attempted to fetch in a single stage. Higher value may
+    be used if system works in reliable environment and the responsiveness is less important then average answer
+    time, it will require more memory reserve though. Decreasing this value may have a positive effect if
+    there are lots of nodes in system and calculations are relatively heavy for each of splits. If the number is too
```

Comment:
`calculations are relatively heavy for each of splits.` -> `processing of each split is relatively expensive.`?